interface/utils/language/es.ts

```diff
@@ -0,0 +1,172 @@
+			commonPassword: "Esta contraseña está en la lista de las 1000 contraseñas más comunes. ¡Por favor escoja una contraseña más segura!",
+		},
+	},
+
+	confirm: {
+		welcomeBack: "¡Bienvenido de nuevo!",
+		confirmPassword: "Confirmar Contraseña",
+		confirmPasswordText: "Por favor ingrese su contraseña para continuar.",
+		password: "contraseña",
+		forgotPassword: "¿Has olvidado tu contraseña?",
+		forgotPasswordText: "Tus códigos están protegidos por tu contraseña. Si olvidó su contraseña no podrá acceder a sus códigos.",
+		dialog: {
+			wrongPassword: "¡Contraseña incorrecta! \n\n¡Inténtalo de nuevo!",
+		},
+	},
+
+	codes: {
+		importCodes: "Importa tus códigos 2FA",
+		importCodesText: "Importe sus códigos 2FA existentes en la página Importar.",
+		importCodesButton: "Importar códigos",
+		noSearchResultsFound: "No se encontraron resultados de búsqueda",
+		noSearchResultsFoundText: "No se encontraron resultados de búsqueda para",
+		dialog: {
+			noSaveFileFound: "No se encontró ningún archivo guardado. \n\n¡Vaya a los códigos o a la página de importación e importe sus códigos!",
+			codesImported: "Códigos importados. \n\nPuedes editar tus códigos en la página de edición..",
+		},
+	},
+
+	import: {
+		supportedTypes: "Tipos de 2FA admitidos",
+		totpQRCode: "Código QR TOTP",
+		totpQRCodeText: "Un código QR TOTP es lo que se encuentra principalmente en todas partes, si desea configurar 2FA. Consta de 6 dígitos que cambian cada 30 segundos.",
+		instructions: "Instrucciones",
+		googleAuthQRCode: "Código QR del autenticador de Google",
+		googleAuthQRCodeText: "Si está utilizando Google Authenticator, puede exportar todos sus códigos existentes e importarlos a Authme.",
+		chooseImportMethod: "Elija el método de importación",
+		importFromImage: "Importar desde imagen",
+		importFromImageText: "Elija una imagen que contenga un código QR compatible.",
+		chooseImageButton: "Elige imagen",
+		enterSecretManually: "Introduzca una clave de configuración",
+		enterSecretManuallyText: "Ingrese una clave secreta TOTP y un nombre.",
+		enterSecretManuallyButton: "Configurar una clave",
+		screenCapture: "La captura de pantalla",
+		screenCaptureText: "Capture un código QR 2FA desde su pantalla.",
+		screenCaptureButton: "Captura de pantalla",
+		webcam: "Webcam",
+		webcamText: "use su Webcam para escanear un código QR 2FA.",
+		webcamButton: "Use webcam",
+		authmeFile: "Archivo de Authme",
+		authmeFileText: "Importe todos los códigos de un archivo de importación de Authme existente.",
+		authmeFileButton: "Elija el archivo",
+		// html dialogs
+		captureScreenTitle: "Importación de pantalla de captura",
+		captureScreenWaiting: "Esperando un código QR...",
+		manualEntry: "Configurar una clave",
+		manualEntryText: "Ingrese un nombre y una clave secreta 2FA.",
+		manualEntryName: "Nombre (requerido)",
+		manualEntrySecret: "Secreto (obligatorio)",
```

Comment:
Whats the difference between requerido and obligatorio? You should only use one

Reply:
Depending on the situation, you may prefer "requerido" (which means required) over "obligatorio" (which means obligatory).